tests/models/swinv2/test_modeling_swinv2.py

```diff
@@ -381,6 +381,9 @@ def test_initialization(self):
                         msg=f"Parameter {name} of model {model_class} seems not properly initialized",
                     )
 
+    def test_pt_tf_model_equivalence(self):
+        pass
```

Comment:
This shouldn't be added in case there's no TF implementation.

Reply:
If I don't overide this function, this test will fail in CI and I can't remove it as its coming from `tests/test_modeling_common.py`.
Is their any other solution for this?

Reply:
This can be removed now!

Reply:
Fixed.